src/evaluate-conditions.js

```diff
@@ -69,6 +69,8 @@ export default function evaluate (model, value, getPreviousValue) {
       })
       if (itemSchemas.length > 1) {
         retModel.items = {anyOf: itemSchemas}
+      } else if (itemSchemas.length === 0) {
+        retModel.items = model.items
```

Comment:
`model.items` should probably be run through the `evaluate` function to remove any conditions further down the rabbit hole (that would make this line `retModel.items = evaluate(model.items, undefined, getPreviousValue)`